WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -35,12 +36,28 @@ export default function Delegates() {
     [delegateGroups],
   );
 
-  const [activeRegion, setActiveRegion] = React.useState();
-  const [adminMode, setAdminMode] = React.useState(false);
+  const [hash, setHash] = useHash();
+
+  const SelectedComponent = React.useMemo(() => {
+    const selectedRegionIndex = delegateRegions.findIndex((region) => region.id === +hash);
```

Comment:
Why do we have the '+' sign?

Reply:
Because `+` is a short version of `parseInt`. I used region.id as hash and I need to parse it before checking if it equals to `region.id`.

Reply:
Does this do proper error handling @maxidragon?

Reply:
@coder13 yes, look at lines 43-45
```js
if (selectedRegionIndex === -1) {
      setHash(delegateRegions[0]?.id);
 }
```